Add missing cli parameter to curl command

gijswijs opened the pull request:
To see the returned HTTP headers of an API request using your app's client ID and secret,
you need to add the `-I` parameter.

gijswijs committed:
Add missing cli parameter to curl command

To see the returned HTTP headers of an API request using your app's client ID and secret,
you need to add the `-I` parameter.

ramyaparimi commented:
@gijswijs The PR has a lot more changes than adding `-I` to the curl command. Please revert back all the other changes and I will triage this for a review 💖 .

gijswijs committed:
Add missing cli parameter to curl command

To see the returned HTTP headers of an API request using your app's client ID and secret,
you need to add the `-I` parameter.

gijswijs committed:
Merge branch 'curl-parameter' of github.com:gijswijs/docs into curl-parameter

gijswijs commented:
@ramyaparimi I see my Prettier formatter had a field day with that file. I forgot to turn it off.
I've reverted the commit, and added a new one with only that single change.  I can make a different PR if that's easier. Just let me know.

gijswijs committed:
Add missing cli parameter to curl command

To see the returned HTTP headers of an API request using your app's client ID and secret,
you need to add the `-I` parameter.

gijswijs committed:
Merge branch 'curl-parameter' of github.com:gijswijs/docs into curl-parameter
